services/harmony/app/models/work-item.ts

```diff
@@ -51,6 +51,9 @@ export default class WorkItem extends Record implements WorkItemRecord {
   // The status of the operation - see WorkItemStatus
   status?: WorkItemStatus;
 
+  // The sub-status of the operation - see WorkItemSubStatus
+  subStatus?: WorkItemSubStatus;
```

Comment:
nit: I think we have coding standard to use underscores in new column names. i.e. `sub_status`. 
I commented above that the `sub_status` name is not consistent in db definition vs migration. If you want to go with `subStatus`, it is fine with me as long as we make it consistent.

Reply:
For some reason I thought it could be snake case in the dB, but camel case in the models. I see now that we are not doing  that (maybe it doesn't work). I'll make it snake case here.